decidim-verifications/config/locales/en.yml

```
@@ -79,7 +79,10 @@ en:
     events:
       verifications:
         verify_with_managed_user:
-          notification_title: The participant <a href="%{resource_path}">%{resource_title}</a> has tried to verify herself with the data of the managed participant <a href="%{managed_user_path}">%{managed_user_name}</a>
+          email_intro: The participant <a href="%{resource_path}">%{resource_title}</a> has tried to verify herself with the data of the managed participant <a href="%{managed_user_path}">%{managed_user_name}</a>.
```

```suggestion
          email_intro: The participant <a href="%{resource_path}">%{resource_title}</a> has tried to verify themself with the data of the managed participant <a href="%{managed_user_path}">%{managed_user_name}</a>.
```
Lately, we try to speak in the generic "they". See #8684 and #6678